pwndbg/commands/got.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import subprocess
+import struct
+
+import pwndbg.commands
+import pwndbg.enhance
+import pwndbg.chain
+import pwndbg.which
+
+from pwndbg.color import *
+
+
+@pwndbg.commands.Command
+@pwndbg.commands.OnlyWithFile
+def got():
+    '''
+    Show the state of the Global Offset Table
+    '''
+
+    local_path = pwndbg.file.get_file(pwndbg.proc.exe)
+
+    for program in ['checksec', 'checksec.sh']:
+        program = pwndbg.which.which(program)
+        if program:
+            argv = [program, "--file" , local_path]
+            out = subprocess.check_output(argv).decode('utf-8')
+            break
+    else:
+        print('Could not find checksec or checksec.sh in $PATH.')
```

Comment:
yes, definitely a good idea ( I had some problem to re-use the function, but probably I made something wrong, I'm going to fix it right now ). I will make the function checksec return an empty string if something went wrong and later check this value inside my function.